docs/getting-started.md

```diff
@@ -1,100 +1,29 @@
+## Use Open Data Hub and JupyterHub instances
 
-### **Set Up AICoE Continuous Integration (CI)**
+* Sign in to JupyterHub [here](https://jupyterhub-opf-jupyterhub.apps.smaug.na.operate-first.cloud/)
+* Watch an intro [tutorial](https://www.youtube.com/watch?v=iI_-lqi3vP4&list=PL8VBRDTElCWpneB4dBu4u1kHElZVWfAwW&index=3) on how to get started.
+* Learn how to monitor your resources [here](https://youtu.be/1zZ-7mdxOWU).
 
-[AICoE-CI](https://github.com/aicoe/aicoe-ci#getting-started) can aid various aspects of a development workflow such as running pre-commit checks, build checks, triggering pipelines to build images, etc.
+## Join the Community! 
 
-For more information on AICoE CI and how to set it up for your project, you can follow the documentation [here](https://github.com/aicoe/aicoe-ci#getting-started).
-
-### **Create Project Boards and Issues**
-
-You can create Issues to highlight new features, bugs and break upcoming goals of a project into smaller chunks. You can create GitHub [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues) and include [Task Lists](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-task-lists) for each task that you plan to work on.
-
-Here is an [example Issue](https://github.com/aicoe-aiops/ocp-ci-analysis/issues/279) that we created to highlight a certain task for our project.
-
-We use [project boards](https://docs.github.com/en/issues/organizing-your-work-with-project-boards/managing-project-boards/about-project-boards) to organize the tasks within the team.
-
-Once your project board is created, begin to create issues and populate the board with them
-* Create an issue for every task you plan to work on
-    * Break issues down into smaller pieces if they cannot be completed in a single sprint
-    * Be specific in the description of the issue 
-    * Tag other related issues with a ‘#’ as necessary
-    * Provide acceptance criteria for when the issue can be closed 
-    * Assign yourself or your teammates who you plan to work with on the task to the issue
-
-* Add the issues to the *New* column of you project board
-* When you're almost ready to begin working on an issue, move it to the *To Do* column
-* Once you start work on an issue, move it to the *In Progress* column
-
-For more information of the remainder of our GitHub workflow, please visit [this page][10]. 
-
-### **Develop on Operate First Jupyterhub**
-
-The following resources are available for use to develop your project on the Operate First JupyterHub instance:
-
-* **Start developing on the Operate First JupyterHub environment**
-    - View the [video tutorial](https://www.youtube.com/watch?v=iI_-lqi3vP4&list=PL8VBRDTElCWpneB4dBu4u1kHElZVWfAwW&index=3)
-    - Access the public [JupyterHub instance][11] on the Massachusetts Open Cloud (MOC) by selecting *operate-first* authentication and then signing in with your GitHub account
-
-* **How to Monitor JupyterHub workloads using Grafana dashboards** 
```

Comment:
this seems like an important piece of info. should be a part of https://github.com/aicoe-aiops/data-science-workflows/pull/82

Reply:
yes. I agree. it can be moved to a different doc